smithy-swift-codegen/src/main/kotlin/software/amazon/smithy/swift/codegen/integration/serde/formurl/FormURLEncodeCustomizable.kt

```diff
@@ -10,4 +10,5 @@ import software.amazon.smithy.model.shapes.Shape
 interface FormURLEncodeCustomizable {
     fun alwaysUsesFlattenedCollections(): Boolean
     fun customNameTraitGenerator(memberShape: Shape, defaultName: String): String
+    fun shouldSerializeEmptyLists(): Boolean
```

Comment:
You could add default implementation here
```
    fun shouldSerializeEmptyLists(): Boolean {
        return true
    }
```
as in all cases it is `true`. ie for EC2Query and AWSQuery

Reply:
Will do if that's preferable